package.json

```diff
@@ -16,7 +16,7 @@
   },
   "scripts": {
```

Comment:
Scripts were already taken care of: https://github.com/meanjs/mean/commit/5a91d7d3a0e7204cd8453129821da58a45d9bd60